api/server/routes/agents/actions.js

```diff
@@ -1,13 +1,20 @@
 const express = require('express');
 const { nanoid } = require('nanoid');
-const { actionDelimiter } = require('librechat-data-provider');
+const { actionDelimiter, SystemRoles } = require('librechat-data-provider');
 const { encryptMetadata, domainParser } = require('~/server/services/ActionService');
 const { updateAction, getActions, deleteAction } = require('~/models/Action');
 const { getAgent, updateAgent } = require('~/models/Agent');
 const { logger } = require('~/config');
 
 const router = express.Router();
 
+// If the user has ADMIN role and the adminCanEditActions is enabled in assistants config 
+// then action edition is possible even if not owner of the assistant
+const isAdmin = (req) => {
+  const adminCanEditActions = req.app.locals?.assistants?.adminCanEditActions ?? false;
```

Comment:
I would rather not add another config for this, just make it the default behavior

Reply:
I thought one of the reasons it was not done was for security reasons to protect the eventual security model of the action.
Would you rather I remove the configuration or set it true by default?

Reply:
I have reverted the adminCanEditActions configuration.
Also I pushed some fixes for special cases:

1. when the admin edits an existing action it should not change the user stored with it ( else the owner will lose access to it and not see it anymore )
2. when the admin adds an action to an existing assistant the action is created with the user of the assistant, not the user of the admin, to avoid the same issue